homeassistant/components/prosegur/config_flow.py

```diff
@@ -95,15 +93,13 @@ async def async_step_reauth_confirm(self, user_input=None):
                 errors["base"] = "cannot_connect"
             except InvalidAuth:
                 errors["base"] = "invalid_auth"
-            except Exception as exception:  # pylint: disable=broad-except
-                _LOGGER.exception(exception)
```

Comment:
We normally log unknown exceptions with the stack strace.

Reply:
can't find any example from existing components doing stack strace, could you point me one ? Thanks!

Reply:
It's in our scaffold:
https://github.com/home-assistant/core/blob/d40012f110ced64afa0e7115b66eed879c078bcf/script/scaffold/templates/config_flow/integration/config_flow.py#L92-L94

Reply:
I was thinking about something different.... but ok :)